gate-web/gate-web.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
 
   implementation "com.netflix.spinnaker.kork:kork-plugins"
   implementation "com.netflix.spinnaker.kork:kork-web"
+  implementation "com.netflix.spinnaker.kork:kork-retrofit"
```

Comment:
I think the original intention with this module was that it should be able to be a runtimeOnly dependency (maybe part of kork-runtime)